batched/dense/src/KokkosBatched_Gesv.hpp

```diff
@@ -63,11 +63,18 @@ struct Gesv {
 
 template <typename ArgAlgo>
 struct SerialGesv {
-  template <typename MatrixType, typename VectorType>
+  template <typename MatrixType, typename XVectorType, typename YVectorType>
```

Comment:
Ideally the TeamGesv and TeamVectorGesv versions would have the same interface, but that can happen later